contribute/views.py

```diff
@@ -1,9 +1,34 @@
+
+
 from django.views.generic import TemplateView
 
+from user.models import Profile, UserLanguage
+from word.models import Tag, Word, WordVersion
+
 
 class ContribView(TemplateView):
     template_name = 'contribute/main.html'
 
+    def get(self, request, *args, **kwargs):
+        context = self.get_context_data(**kwargs)
+
+        try:
+            user_profile = Profile.objects.get(user=request.user)
+        except TypeError:
+            user_profile = None
+
+        if user_profile:
+            user_language = UserLanguage.objects.get(user=user_profile)
+            if user_language.is_moderator:
```

Comment:
This could be shortened
`context["user_moderator"] = bool(user_language.is_moderator)`

Reply:
now I complicated a little bit more the code because of the many languages and I think I can not use that :(